### PR TITLE
Setting up the database for missing samples btw ul16/18 and ul17

### PR DESCRIPTION
**Pull request by pgunnell**
Printing in html not yet implemented!

### CODE REVIEW

File: update_ul17based.py [hunk 1]
```diff
@@ -51,6 +76,26 @@
...
 
+                present_ul18 = True
+
+        if  not present_ul18:
+
+            chained_request_id = request_ul17['member_of_chain'][0]
+            chained_request = mcm.get('chained_requests', query='prepid=%s' %chained_request_id)
+            root_id = chained_request[0]['chain'][0]
+
+            cursor.execute('INSERT INTO missing_ul VALUES (?, ?, ?, ?, ?, ?, ?)',
```
Comment (justinasr) [inline update_ul17based.py:87]: This script is inserting values into the table every time it runs. Doesn't this duplicate data in the database?

**Comment by pgunnell** [comment]
What I would like to do next is to print in the html page of campaign groups lines from missing_ul table like this:

Short name  - Dataset name - 'Not in the system' - chain - number of events - prepid_in_ul_17

**Comment by justinasr** [comment]
> What I would like to do next is to print in the html page of campaign groups lines from missing_ul table like this:
> 
> Short name - Dataset name - 'Not in the system' - chain - number of events - prepid_in_ul_17

As a separate page or in the main samples page?

**Comment by pgunnell** [comment]
> > What I would like to do next is to print in the html page of campaign groups lines from missing_ul table like this:
> > Short name - Dataset name - 'Not in the system' - chain - number of events - prepid_in_ul_17
> 
> As a separate page or in the main samples page?

in the main samples page if possible

something like here:

https://cms-pdmv-dev.cern.ch/samples/campaign_group/RunIISummer19UL17/JME

(at the bottom of the page)

**Pull request merged**